### PR TITLE
front: fix zoom buttons in maps

### LINKED ISSUE #6458: Zoom in button on map brings me to the center of France
### What happened?

I'm viewing Geneva on the map.
image

Clicking the `zoom in` button on the map bring me near Tours, which isn't where i was initally.
image

### What did you expect to happen?

I'm expecting the `zoom in` button to zoom where I am.

### How can we reproduce it (as minimally and precisely as possible)?

1. Go to the map on France infra
2. Anywhere on the map
3. Zoom in

### What operating system, browser and environment are you using?

- **Browser**: Chrome
- **OS**: W11
- **Env**: stagin internal

### OSRD version (top right corner `Account` button > `Informations`)

f062f1f

### PR DESCRIPTION
**Pull request by achrafmohye**
close #6458

### CODE REVIEW

File: front/src/common/Map/Buttons/MapButtons.tsx [hunk 1]
```diff
@@ -101,25 +97,28 @@ export default function MapButtons({
...
-  }, [openedPopover, isOpen]);
+  }, [openedPopover, isOpen, closeFeatureInfoClickPopup]);
 
   // Close the Popover when opening modal
   useEffect(() => {
     setOpenedPopover(undefined);
   }, [isOpen]);
 
-  useOutsideClick(mapButtonsRef, () => setOpenedPopover(undefined));
+  // Update the viewport when the props change
```
Comment (kmer2016) [inline front/src/common/Map/Buttons/MapButtons.tsx]: Really need this comment ?

**Review by kmer2016 — COMMENTED** [review]
a very small comment. Well done to improve MapButton perf by using usecallback.

File: front/src/common/Map/Buttons/MapButtons.tsx (hunk 1 above)
Comment (achrafmohye) [reply front/src/common/Map/Buttons/MapButtons.tsx]: Non, je l’enlève donc :)

**Comment by kmer2016** [comment]
The CI broke due to the format of your commit messages. Please follow the rules described [here](https://osrd.fr/fr/docs/guides/contribute/contribute-code/commit-style/) to correctly format your commit messages.

**Review by Math-R — APPROVED** [review]
lgtm !

**Pull request merged**